### DISCUSSION TITLE
Shortcuts are disabled and I can't change it

### DISCUSSION THREAD

**Discussion by ralphsiegert** [rxhanson/Rectangle#1475] [Q&A] [answered]
I didn't find an answer to that. Unfortunately, after installing Rectangle I don't get the shortcuts working. They are just disabled. 

<img width="261" alt="Bildschirmfoto 2024-09-21 um 17 35 10" src="https://github.com/user-attachments/assets/a373e620-294a-4cfc-8628-f67447d64ad1">

Macbook Air M2, Sonoma 14.6.1. 

Can you please support me with the right hint? Thanks.

**Comment by rxhanson** [comment, marked as answer]
The menu items don't look disabled. The shortcuts are always shown as a lighter color, that's macOS design. If the titles were grayed out as well, then that would point to a common issue where the Accessibility permissions get out of sync in macOS.

As a quick sanity check, are you sure that you're using the proper modifier keys (control + option)?

We can reset the accessibility permissions by
1. Quitting Rectangle
2. In System Settings -> Privacy & Security -> Accessibility, disable then remove Rectangle.
3. Relaunch Rectangle and provide accessibility permissions again

**Reply by ralphsiegert** [reply]
Thank you so much, Ryan. The bug was sitting in front of the computer. I really used the wrong key combination, I was interpreting the first as the shift key. Together with the grayed color, I wasn't even looking for the issue on my side. 
I am very sorry. and closing here.
